### DISCUSSION TITLE
Table

### DISCUSSION THREAD

**Discussion by borisschelcher** [grafana/grafana#63313] [Dashboards, Panels & Visualizations]
Is it planned to add to the table panel the line breaker like the old one ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
